src/shared_types.rs

```diff
@@ -479,9 +316,11 @@ impl<'de> Deserialize<'de> for NonNegativeIntegerOrFloat {
 
 #[cfg(test)]
 mod tests {
-    use super::*;
     use serde_test::{assert_tokens, Token};
 
+    use super::*;
+    use crate::*;
```

Comment:
does this make `use super::*` redundant?

Reply:
Not entirely, but I have expanded the imports now.